DuggaSys/diagram.js

```diff
@@ -755,11 +758,14 @@ var movingContainer = false;
 var isRulerActive = true;
 
 //Grid Settings
-const gridSize = 50;
-const origoWidth = 2;
-var snapToGrid = false;
-var randomidArray = []; // array for checking randomID
-var errorMsgMap = {};
+var gridSettings = {
+
+    gridSize: 50,  
+    origoWidth: 2,
+    snapToGrid:false,
+    randomidArray: [], // array for checking randomID
+    errorMsgMap: {},
+};
```

Comment:
Call it just settings or something like that since it contains more than just grid settings. Then put `zoomPower` and `isRulerActive` into it.